### PR TITLE
Add User Stories for `Add`, `View` and NFRs 

### PR DESCRIPTION
**Pull request by alson001**

### CODE REVIEW

**Review by JamesLiuZX — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -285,14 +285,60 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+  Use case ends.
+
+* 2b. The storage file is invalid.
+
+    * 2b1. Book shows an error message, with a button that displays "file fixed" to check again if the file is valid.
+    * 2b2. Chef rectifies/fixes error with the file.
+    * 2b3. Chef clicks the "file fixed" button
+
+      Use case resumes at step 2.
```
Comment (JamesLiuZX) [inline docs/DeveloperGuide.md:309]: `From step 2 `

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -285,14 +285,60 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+      Use case ends.
+
+**Use case: Delete a recipe**
+
+**MSS**
+
+1. Chef requests to list recipes
+2. Book shows a list of ***all*** recipes
+3. Chef requests to delete a specific recipe in the list
```
Comment (JamesLiuZX) [inline docs/DeveloperGuide.md:340]: `delete a specific recipe from the list`

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (SeeuSim) [reply docs/DeveloperGuide.md]: Resolved by [latest commit](https://github.com/AY2223S2-CS2103T-T13-2/tp/pull/37/commits/8150599b3e6db6bfc96e4045e1dd37b6e57faa99)

**Review by SeeuSim — APPROVED** [review]
LGTM

**Pull request merged**